GameDefinition/Content/ItemKind.hs

```diff
@@ -215,7 +215,7 @@ harpoon2 = harpoon
   { ifreq    = [("common item", 2), ("harpoon", 2)]
   , iweight  = 1000
   , idamage  = 10 `d` 1
-  -- , idesc    = ""
+  , idesc    = "A barbed, heavy spear for hunting larger game."
```

Comment:
Now, that is very interesting item concept, but I guess it rather fits a durable melee weapon with range 2 (melee weapons you can reach to distant tiles with are fun in other roguelikes, but are rather painful UI-wise so not implemented in our game). I wonder how to handle a very rare harpoon that is heavier and twice more damaging than another type of harpoon. Perhaps make the first one a fishing harpoon and the second one a whaling harpoon? 

OTOH, your idea made me aware, a harpoon, especially a heavy one, is not very likely to break after a single use. But we can't make it durable, because the player would go into a boring routine of "start every fight with a barrage of the power durable harpoons, mop up with melee weapons". Perhaps the fishing harpoon are rather flimsy and the whaling harpoon is antique and so brittle? Any other ideas are welcome, as always, feel free to edit everything you fancy.

Reply:
What about "A heavy spear, for larger game. The barbed tip breaks off and stays in the victim's flesh."

Reply:
This sounds like the Roman pilum, which was indeed designed to break off, so it's quite plausible something like that could be used against formidable animals as well.  However, pilum was much lighter that what I imagine from your description --- I guess, the heavier spears are too cumbersome for throwing. Would "heavy javelin" make sense? Other than that, it sounds good and doesn't strain the `harpoon` theme --- but we need to remove the `PullActor` effect, since there's no line to pull with. OH, and change the `iname` and the verb, I guess.

Reply:
Looks great.